### ISSUE TITLE
Better Notifications for failed backups 

### ISSUE THREAD

**Issue by musikele** [duplicati/duplicati#3379]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info
- **Duplicati version**: 2.0.3.3_beta_2018-04-02
- **Operating system**: Windows 10 Professional
- **Backend**: SFTP 

## Description

When the backup fails, Duplicati doesn't notify me in any way. 

## Steps to reproduce

1. Start a backup...
2. turn off internet connection to the pc being backupped 
3. duplicati won't show any alert of the backup gone wrong. 

- **Actual result**:
Only the status icon in the system tray becomes red instead of grey, there's no other useful info 

- **Expected result**:
Duplicati shows a notification coming via the Push Notification API in the web browser, or sends an email to a configured access

**Comment by ts678** [comment]
From the web UI, please look over the "Send mail" set of options and similar sets on the categorized listing. Third party software picks up from there. Please see [Android/IOS/Windows Notifications](https://forum.duplicati.com/t/android-ios-windows-notifications/4483) for links and news.
[Reporting options](https://duplicati.readthedocs.io/en/latest/06-advanced-options/#reporting-options) gives explanations of how to use advanced options, and the [forum](https://forum.duplicati.com/) often has discussions. Some of this is a little hard to spot despite general resources like [Settings in Duplicati](https://duplicati.readthedocs.io/en/latest/03-using-the-graphical-user-interface/#settings-in-duplicati).

**Comment by AnderssonPeter** [comment]
While this is true and works like a charm, it won't work if your offline?

**Comment by ts678** [comment]
I think remote notification must be online and isn't queued. I might be mistaken, because I'm rarely offline.
From the original request, was the expected "Expected result" email intended to be sent when back online?

I rarely run Tray Icon, but when I did I noticed Windows 10 giving me Duplicati error notifications. In fact at the same time that the icon turned red, I got a sound, an error message sliding in from the right above the clock, and the self-clearing message persisted in the notification area activated by the bottom right button.

I'm not sure what would happen on other systems, but this look to be quite a complete offline notification.
This is the persisted notification on Windows 10 Professional Version 1803 running Duplicati 2.0.3.9 canary:

![image](https://user-images.githubusercontent.com/40771998/47025793-28ed2400-d132-11e8-8352-fe999f380fe0.png)
